simplemma/rules.py

```diff
@@ -160,7 +145,7 @@ def apply_de(token: str, greedy: bool = False) -> Optional[str]:
 ENGLISH_S_ENDING = re.compile(r"(dom|ism|ist|ment|nce|ship|tion|um)s$")
 
 
-def apply_en(token: str) -> Optional[str]:
+def apply_en(token: str, greedy: bool = False) -> Optional[str]:
```

Comment:
We could remove `greedy` here and thereafter, it's not used and I don't think it will be

Reply:
The reason to have is so all the method have the same typings in the dictionary.
Otherwise, there are errors.